src/analyserbeats.cpp

```diff
@@ -177,7 +200,7 @@ bool AnalyserBeats::loadStored(TrackPointer tio) const {
     }
 }
 
-void AnalyserBeats::process(const CSAMPLE *pIn, const int iLen) {
+void AnalyserBeats::process(const CSAMPLE* pIn, const int iLen) {
```

Comment:
With these types of changes, I thought putting the * on the type helps us know that the variable is a pointer-to-a-thing instead of the thing.